### DISCUSSION TITLE
Discussion about adding external link, logo, visibility to Zitadel applications

### DISCUSSION THREAD

**Discussion by doncicuto** [zitadel/zitadel#5584] [Contributors]
I've some UI questions about the desired location and behavior for issue #4586 

First question. Where should the external url, visibility and light/dark logos fields be placed in the new app wizard?

I suggest a new Dashboard step (screenshot not showing dark/light icon fields right now

![image](https://user-images.githubusercontent.com/30386061/229284799-8a770022-74d9-41ce-9830-8ea4b2d76707.png)

**Comment by doncicuto** [comment]
Second question. In the devmode view (wizard not used), should external url, visibility and icons be shown in that view, or do you want me to add a check that shows those fields to developers.

![image](https://user-images.githubusercontent.com/30386061/229284918-025534e0-edf4-410c-b5d7-f0f441875589.png)

**Comment by hifabienne** [comment]
@juergrinaldi can you have a look at this? If you have any questions let me know I can help you.

**Comment by hifabienne** [comment]
@peintnermax can you please have a look at this?

**Comment by peintnermax** [comment]
I'd not place this fields in the app create wizard but place it on the detail view of the application..
The behavior would then be the same as for users and keeps the creation part clean with the required fields only. Whats your opinion about that? @juergrinaldi

**Reply by juergrinaldi** [reply]
I agree. I'd rather keep the the 'create process' slick and stick to the essentials.

IF we want to add them to the creation process add a 'skip' button, so the user gets the info that it is an optional step.
But as said: I would not add it for now. Instead just add a new submenu entry and keep it all there for the user to change after creating the app.

Random input that came to my mind: We could add a little hint for the user to this section as this:
After the user created a new app, we could show a toast message like _'You can customize your app **here**.'_

What do you guys think?

**Reply by doncicuto** [reply]
Hi @peintnermax  and @juergrinaldi, awesome. I'll prepare new screenshots to reflect your suggestions. Hope you don't mind but I'll have to ask you a little more before I create my final PR

**Reply by juergrinaldi** [reply]
@doncicuto don't worry. just drop any questions here. We're glad for your help and to have you as a contributor. 🦒 😃
